timetracker/entries/urls.py

```diff
@@ -4,9 +4,10 @@
 
 
 urlpatterns = [
-    url(r'^clients/$', views.clients, name='client-list'),
+	url(r'^$', views.RootRedirectView.as_view(), name='root'),
```

Comment:
Some weirdness with sublime text going on here. I caught it doing this once but missed this one. It is set to edit python code with not hard tabs. (I normally use PyCharm which to my knowledge never gets this wrong)